imageresizerapp/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,5 @@
+Uninstall-ChocolateyPackage `
+  -PackageName 'imageresizerapp' `
+  -FileType 'EXE' `
+  -Silent '/quiet /norestart /uninstall' `
+  -File (((Get-UninstallRegistryKey 'Image Resizer for Windows').UninstallString -split '"')[1])
```

Comment:
The WiX bootstrapper also specifies a product code...somewhere. I guess if I ever localize the setup, I'll figure out how to look this up using it.